.github/workflows/reusable-workflow-lint.yml

```
@@ -0,0 +1,24 @@
+name: Lint GitHub Actions workflows
+on:
+  workflow_call:
+
+permissions: {}
+
+jobs:
+  actionlint:
+    name: Actionlint
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    timeout-minutes: 1
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          persist-credentials: false
```

```suggestion
          persist-credentials: false
          show-progress: ${{ runner.debug == '1' && 'true' || 'false' }}
```

We have this in other workflows calling `checkout` to limit the output when debug mode is disabled.